src/@types/types.ts

```diff
@@ -49,3 +54,9 @@ export interface CentreonLoginResult {
     token: string;
   };
 }
+
+export type tFilter = SavedFilter & { type: SelectableValue<string> & { valid?: boolean } };
+
+export type TypeFilter = SelectableValue<MBIResourceType>;
+
+export type strOrArrStr = undefined | string | Array<undefined | string>;
```

Comment:
Same remark concerning the PascalCase naming, and besided, we should make the name clearer